packages/kbn-test/src/kbn_client/kbn_client_requester.ts

```diff
@@ -149,18 +150,17 @@ export class KbnClientRequester {
         const requestedRetries = options.retries !== undefined;
         const failedToGetResponse = isAxiosRequestError(error);
 
-        if (isIgnorableError(error, options.ignoreErrors)) {
-          return error.response;
-        }
+        if (isIgnorableError(error, options.ignoreErrors)) return error.response;
+
+        let errorMessage: string = '';
+        const redactedUrl = redactUrl(description.split(' ')[1]);
```

Comment:
__question__: are we sure `description` always comes with space and we won't have error here `Array Out of Bounds`?

Reply:
Not sure, but I can protect it, just in case!

Reply:
That's a great point!